lib/utils.js

```diff
@@ -21,4 +21,9 @@ utils.startAtZero = function startAtZero(obj, key) {
   }
 };
 
+utils.isAbsolutePath = require('path').isAbsolute;
+if (!utils.isAbsolutePath) {
+  utils.isAbsolutePath = require('path-is-absolute');
```

Comment:
So it is only supposed to work if I am running windows, which I am not, so this may be ok.

see https://github.com/sindresorhus/path-is-absolute/issues/1